src/universal/modules/outcomeCard/components/OutcomeCardFooter/OutcomeCardFooter.js

```diff
@@ -29,6 +30,16 @@ const LoadableAssignMenu = Loadable({
   timeout: MAX_WAIT_TIME
 });
 
+const LoadableAssignTeamMenu = Loadable({
```

Comment:
Just out of curiosity, what's your razor for deciding when to dynamically import a module vs include it in the bundle?

Reply:
The razor is....
Make EVERYTHING a loadable. then, it's up to webpack to decide how to split up the packages.

Before react-loadable, the razor was; "is the content visible?" menus are a great example, because chances are, you won't click that GH menu, so why should your browser spend time parsing it?